src/standalone.ts

```diff
@@ -67,11 +55,7 @@ async function downloadAndCheckVersion(
 
   // only download addon if its new or the id has changed
   if (!oldRelease || oldRelease.id !== id) {
-    const release = await downloadStandalone(addon, host_url, id)
-
-    if (!release) {
-      throw new Error(`no release asset found for addon ${addon.package.name}`)
```

Comment:
This just assumed that the release can only be undefined if the addon is an archive without valid assets, this is now directly handled in the function handling archives.